subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/TestDescriptorInternal.java

```diff
@@ -36,18 +36,7 @@
     @Nullable
     Object getOwnerBuildOperationId();
 
-    /**
-     * The name for display. It may be the same as or different from {@link #getName()}
-     *
-     * @return the name for display.
-     */
-    String getDisplayName();
```

Comment:
I'm thinking about moving these changes to a separate PR so we can better promote it in the release notes. WDYT?

Reply:
Makes sense. 👍

Reply:
This is the change to move getDisplayName and getClassDisplayName from an internal class to the TestDescriptor? It does seem like a separate issue.